lib/wellies_web/handlers/response.ex

```diff
@@ -0,0 +1,17 @@
+defmodule WelliesWeb.ResponseHandler do
+
+  def get_body(response) do
```

Comment:
You could actually do the pattern matching in the function's signature, instead of inside:
```
def get_body({:ok, {{'HTTP/1.1', 200, 'OK'}, _headers, body}}), do: Jason.decode!(body)
```
Do take into account other scenarios like `def get_body(_), do: ...`